README.md

```
@@ -5,195 +5,223 @@
+- Visual before/after comparison for each change
+- CSS selector and file location
+- Original and updated WCAG levels
+- Color swatches showing differences
 
-# Mix formats in a pair
-pair = ColorPair("rebeccapurple", "#ffffff")
-print(pair.contrast_ratio)  # 6.95
-print(pair.wcag_level)      # AA
+**Console output:**
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)